packages/services/api/src/modules/audit-logs/module.graphql.ts

```diff
@@ -0,0 +1,236 @@
+    user: AuditLogUserRecord!
+    organizationId: String!
+    inviteeId: String!
+    inviteeEmail: String!
+  }
+  type UserRemovedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    removedUserId: String!
+    removedUserEmail: String!
+  }
+
+  type OrganizationSettingsUpdatedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    updatedFields: JSON!
+  }
+
+  type OrganizationTransferredAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    newOwnerId: String!
+    newOwnerEmail: String!
+  }
+
+  type ProjectCreatedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    projectId: String!
+    projectName: String!
+  }
+
+  type ProjectSettingsUpdatedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    projectId: String!
+    updatedFields: JSON!
+  }
+
+  type ProjectDeletedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    projectId: String!
+    projectName: String!
+  }
+
+  type TargetCreatedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    projectId: String!
+    targetId: String!
+    targetName: String!
+  }
+
+  type TargetSettingsUpdatedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    projectId: String!
+    targetId: String!
+    updatedFields: JSON!
+  }
+
+  type TargetDeletedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    projectId: String!
+    targetId: String!
+    targetName: String!
+  }
+
+  type SchemaPolicySettingsUpdatedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    projectId: String!
+    updatedFields: JSON!
+  }
+
+  type SchemaCheckedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    projectId: String!
+    targetId: String!
+    schemaSdl: String!
+  }
+
+  type SchemaPublishAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    projectId: String!
+    targetId: String!
+    schemaName: String!
+  }
+
+  type SchemaDeletedAuditLog implements AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+    projectId: String!
+    targetId: String!
+    schemaName: String!
```

Comment:
`serviceName: String`

Reply:
If you run this when you delete a service, then please rename the GraphQL type to be `ServiceDeletedAuditLog`